### PR TITLE
[UNR-2731] Disable example project CI on 4.22

### PR DESCRIPTION
**Pull request by mironec**
As per discussion, we do not support the example project on 4.22 anymore.

### CODE REVIEW

**Comment by mironec** [comment]
Link to build: https://buildkite.com/improbable/unrealgdkexampleproject-nightly/builds/438#351e01ed-8c2c-4c05-bc11-9a2627a0ecc2

File: ci/generate-pipeline-steps.sh [hunk 1]
```diff
@@ -20,11 +20,16 @@ done
 # We retrieve these engine versions from the unreal-engine.version file in the UnrealGDK repository.
 # The steps are based on the template in nightly.template.steps.yaml.
 
+# Default to only testing the first version listed in the unreal-engine.version file
+MAXIMUM_ENGINE_VERSION_COUNT_LOCAL=${MAXIMUM_ENGINE_VERSION_COUNT:-1}
```
Comment (jessicafalk) [inline ci/generate-pipeline-steps.sh]: usually I would quotes around them, but not sure if that will break line 30?

File: ci/generate-pipeline-steps.sh [hunk 2]
```diff
@@ -20,11 +20,16 @@ done
...
 if [ -z "${ENGINE_VERSION}" ]; then 
-    echo "Generating build steps for each engine version listed in unreal-engine.version"
+    echo "Generating build steps for the first $MAXIMUM_ENGINE_VERSION_COUNT_LOCAL engine versions listed in unreal-engine.version"
```
Comment (jessicafalk) [inline ci/generate-pipeline-steps.sh]:
```suggestion
    echo "Generating build steps for the first ${MAXIMUM_ENGINE_VERSION_COUNT_LOCAL} engine versions listed in unreal-engine.version"
```

File: ci/generate-pipeline-steps.sh (hunk 1 above)
Comment (mironec) [reply ci/generate-pipeline-steps.sh]: I will try out a build, just in case

**Pull request merged**